### PR TITLE
feat: add terradoc

### PR DESCRIPTION
**Pull request by kassianh**
Recreated README with terradoc, checked documentation for errors or inconsistencies.

### CODE REVIEW

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+
+      - **`folder_iam_member`**
+
+        All attributes of the created `google_folder_iam_member` resource.
+
+      - **`organization_iam_member`**
+
+        All attributes of the created `google_organization_iam_member` resource.
+
+      <!-- all outputs in outputs.tf-->
```
Comment (mariux) [inline README.tfdoc.hcl:385]:
```suggestion
```

**Review by mariux — DISMISSED** [review]
please do not add html comments into tfdoc ;)

**Review by mariux — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -40,8 +44,6 @@ This module implements the following terraform resources
 
 and supports additional features of the following modules:
 
-- [mineiros-io/terraform-google-service-account-iam](https://github.com/mineiros-io/terraform-google-service-account-iam)
```
[the remark this answers is not in the document]
Comment (kassianh) [reply README.md:43]: Ah, i think this is a readme2tfdoc issue

**Review by mariux — CHANGES_REQUESTED** [review]
please rerender markdown

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**